stylesheets/blog.css

```
@@ -0,0 +1,20 @@
+.blog__inner{
+    padding-bottom:120px;
+}
+
+.blog__entry{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    align-items: center;
+    margin: 60px;
+}
```

Tip: You can attach `padding` and `margin` to the same element:

```suggestion
.blog__entry{
    display: grid;
    grid-template-columns: 1fr 1fr;
    align-items: center;
    margin: 60px;
    padding-bottom:120px;
}
```

---

(If you only wanted it to apply to a particular `blog__entry` you could also look at using something like [`:first-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child) or [`:last-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child).